app/models/user.rb

```diff
@@ -1,7 +1,13 @@
 class User < ApplicationRecord
+  attr_accessor :remember_token
+  before_save :downcase_email
+  
   has_many :bills, dependent: :destroy
+
   has_many :rates, dependent: :destroy
+
   has_many :news, dependent: :destroy
+  
   has_many :comments, dependent: :destroy
```

Comment:
không phải. 
quan hệ cho riêng 1 khối. 
callback 1 khối.
before action 1 khối.
.... 
VD:
```suggestion
  attr_accessor :remember_token
  
  before_save :downcase_email

  has_many :bills, dependent: :destroy
  has_many :rates, dependent: :destroy
  has_many :news, dependent: :destroy
  has_many :comments, dependent: :destroy
```